packages/client/wallets/aa/src/api/CrossmintWalletService.ts

```diff
@@ -30,71 +30,6 @@ export class CrossmintWalletService extends BaseCrossmintService {
         );
     }
 
-    async getOrAssignWallet(userIdentifier: UserIdentifier) {
-        return this.fetchCrossmintAPI(
-            "unstable/wallets/aa/ncw",
-            { method: "POST", body: JSON.stringify({ userIdentifier }) },
-            `Error getting or assigning wallet for user: ${JSON.stringify({ userIdentifier })}`
-        );
-    }
-
-    async unassignWallet(userIdentifier: UserIdentifier) {
-        return this.fetchCrossmintAPI(
-            "unstable/wallets/aa/ncw/unassign",
-            {
-                method: "POST",
-                body: JSON.stringify({ userIdentifier }),
-            },
-            `Error unassigning wallet for user: ${JSON.stringify({ userIdentifier })}`
-        );
-    }
-
-    async rpc(walletId: string, deviceId: string, payload: string) {
-        return this.fetchCrossmintAPI(
-            "unstable/wallets/aa/ncw/rpc",
-            {
-                method: "POST",
-                body: JSON.stringify({ walletId, deviceId, payload }),
-            },
-            "Error creating abstract wallet. Please contact support"
-        );
-    }
-
-    async createTransaction(data: any, walletId: string, assetId: string, typedMessage: boolean) {
-        return this.fetchCrossmintAPI(
-            "unstable/wallets/aa/ncw/transaction",
-            {
-                method: "POST",
-                body: JSON.stringify({ data, walletId, assetId, typedMessage }),
-            },
-            `Error creating transaction for wallet: ${walletId}`
-        );
-    }
-
-    async getSignature(txId: string) {
-        return this.fetchCrossmintAPI(
-            `unstable/wallets/aa/ncw/transaction?txId=${txId}`,
-            { method: "GET" },
-            `Error getting signature for transaction: ${txId}`
-        );
-    }
-
-    async getAddress(walletId: string, accountId: number, assetId: string) {
-        return this.fetchCrossmintAPI(
-            `unstable/wallets/aa/ncw/address?walletId=${walletId}&accountId=${accountId}&assetId=${assetId}`,
-            { method: "GET" },
-            `Error getting address for wallet: ${walletId}, account: ${accountId} and asset: ${assetId}`
-        );
-    }
-
-    async getNCWIdentifier(deviceId: string) {
-        return this.fetchCrossmintAPI(
-            `unstable/wallets/aa/ncw?deviceId=${deviceId}`,
-            { method: "GET" },
-            `Error getting NCW identifier for device: ${deviceId}`
-        );
-    }
-
```

Comment:
we're removing this because we don't support fireblocks anymore? We can't just remove this functionality–if we need to add it back in a separate PR then let's make sure that PR exists first but can't merge this in removing Fireblocks